src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
Ah, ok, I hadn't taken it that far in my head, but now that I see it, that makes a lot of sense.

What I would say is, it would probably be better to do this breaking change, sooner rather than later, while the number of external addin's is manageable.